app/controllers/meetings_controller.rb

```diff
@@ -0,0 +1,134 @@
+class MeetingsController < Comfy::Admin::Cms::BaseController
+  before_action :set_meeting, only: %i[ show edit update destroy ]
```

Comment:
ensure that the user can access email, so this controller is locked down via authorization